src/PySpectrometer2.py

```diff
@@ -144,17 +144,40 @@
 cursorX = 0
 cursorY = 0
 def handle_mouse(event,x,y,flags,param):
-	global clickArray
-	global cursorX
-	global cursorY
+	global clickArray, cursorX, cursorY, peak_intensities, wavelengthData
 	mouseYOffset = 160
 	if event == cv2.EVENT_MOUSEMOVE:
 		cursorX = x
 		cursorY = y	
-	if event == cv2.EVENT_LBUTTONDOWN:
+	elif event == cv2.EVENT_LBUTTONDOWN:
 		mouseX = x
 		mouseY = y-mouseYOffset
-		clickArray.append([mouseX,mouseY])
+		if len(peak_intensities) > 0:
```

Comment:
I'm not sure if selecting a peak within +/- 5 pixels is better, or if choosing the closest label is. I've left +/-5 on, but maybe this could also be an option enabled by passable arguments?